gsms.py

```diff
@@ -205,5 +304,38 @@ def main() -> None:
                                 'custom location.')
 
 
+def get_win_path(folderid, user_handle=UserHandle.current):
+    """
+    Function to resolve Windows UUID folders into their absolute path
+
+    Parameters
+    -------
+    folderid: string
+    user_handle: UserHandle
+
+    Returns
+    -------
+    Path as string
+
+    Raises
+    ------
+    PathNotFoundException
+        When a UUID can not be resolved to a path as it is not a windows UUID path
+
+    Examples
+    --------
+    >>> get_win_path("{B4BFCC3A-DB2C-424C-B029-7FE99A87C641}", UserHandle.Common)
+    C:\\Users\\Default\\Desktop
+    """
+    fid = GUID(uuid.UUID(folderid))
+    pPath = ctypes.c_wchar_p()
+    S_OK = 0
+    if _SHGetKnownFolderPath(ctypes.byref(fid), 0, user_handle, ctypes.byref(pPath)) != S_OK:
+        raise PathNotFoundException()
```

Comment:
This exception allows the code to continue execution if raised?

Reply:
Custom exception is removed. WIll raise `Exception` with custom message to identify the issue